decorator_allowed_groups/models/decorator.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2021-Today: GRAP (<http://www.grap.coop/>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api
+from odoo.exceptions import AccessError
+
+
+def allowed_groups(*group_xml_ids):
```

Comment:
The decorator is not a model. IMO it should be moved into decorator_allowed_groups/api.py